api_app/connectors_manager/classes.py

```diff
@@ -64,7 +64,7 @@ def health_check(cls, connector_name: str) -> Optional[bool]:
             url = cc.read_secrets(secrets_filter="url_key_name").get(
                 "url_key_name", None
             )
-            if url is not None:
+            if url and url.startswith("http://"):
```

Comment:
I think it should also be acceptable to add HTTPS support? I mean, You never know right? `if url and url.startswith(("http://", "https://")):` should do it.

Reply:
ah yeah you right 🤦🏻 thanks for pointing this out